### ISSUE TITLE
Adding a blog functionality to `Community Contributions`

### ISSUE THREAD

**Issue by aktdenis** [akash-network/website#263]
Currently, the [Community Contributions](
https://akash.network/community/community-contributions/) section on the Akash Network website serves as a placeholder linking to external content. 

As we talked during wg-website meets we could introduce a blog functionality (mirroring the format and style used in the main blog) to showcase community-written blog posts directly on the Akash website. 

Here is an example of a community member written blog post currently living in the main blog sections:
https://akash.network/blog/the-rise-of-decentralized-compute/.

**Comment by HoomanDigital** [comment]
Yes, this is something we also discussed during the wg-meet.. if there is UI changes, we will share the reference.. thanks

**Comment by HoomanDigital** [comment]
Below is the overview of possible implementation as per discussion and requirements:


![community contributions (1)](https://github.com/akash-network/website/assets/172995301/eac71f32-9cc6-4dd5-98f2-12ed608e605b)

**Comment by ShinySyntax** [comment]
It looks like good.

**Comment by HoomanDgtl** [comment]
his @aktdenis : do we have further guidelines for this issue?

thanks

**Comment by ShinySyntax** [comment]
Could you assign to me this task?
@aktdenis
